diagnostics/mem.go

```diff
@@ -0,0 +1,28 @@
+package diagnostics
+
+import (
+	"encoding/json"
+	"github.com/ledgerwatch/erigon-lib/common/dbg"
+	"net/http"
+)
+
+func SetupMemAccess(metricsMux *http.ServeMux) {
+	metricsMux.HandleFunc("/mem", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
a bit unclear why need separated http endpoint for memory metrics. we already have `/debug/metrics/prometheus` (enabling by flag --metrics)

Reply:
I may want to use it to show it in diagnostics